.github/workflows/ci.yml

```diff
@@ -251,7 +251,9 @@ jobs:
       - run: node --experimental-strip-types ./internal/lsp/lsproto/_generate/generate.mts
 
       - name: Regenerate fourslash tests and update failing test list
-        run: npm run updatefailing
+        run: |
+          npm run convertfourslash
+          npm run updatefailing
```

Comment:
`npm run updatefailing` will run `convertFourslash` (twice), so, not sure why we'd need to add it here.

Reply:
Yeah I was just trying to figure out why the test is failing even though I already committed the file...

Reply:
I see. Yeah, I don't get why it's failing. 🙁